### DISCUSSION TITLE
BF1 [BF]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#398] [Show and tell]
### Channel ID

BF1.bf

### Stream URL (optional)

https://547f72e6652371c3.mediapackage.us-east-1.amazonaws.com/out/v1/7e66f389bec3463dac81e8d55e680431/index.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://547f72e6652371c3.mediapackage.us-east-1.amazonaws.com/out/v1/7e66f389bec3463dac81e8d55e680431/index.m3u8 (black screen)
<img width="1415" alt="image" src="https://user-images.githubusercontent.com/7253922/233812069-3c58bba4-58bd-4826-836e-693228b8efbf.png">
